juniper_official/lldp/lldp-session-state.rule

```diff
@@ -0,0 +1,173 @@
+/*
+ * Detects LLDP (session) state changes and notifies when anomalies are found.
+ */
```

Comment:
Add the requirement ID. Please below rule for reference.
https://github.com/Juniper/healthbot-rules/blob/jcloud/juniper_official/chassis/check-psm-power-usage-state.rule

Reply:
Added Req ID